### ISSUE TITLE
Training Speed

### ISSUE THREAD

**Issue by Lewis-lv0** [skhu101/SHERF#39]
Hi @skhu101 , thanks for your great work! I am trying to train the model on RenderPeople dataset but I found the training will take ~4 days. May I ask what can be the main reason for the training to take such a long time? Here are my guesses: rendering the whole image before calculating the training loss; global feature extractor (StyleGAN); transformer-based feature fusion module; large training steps. 

I would appreciate if I can hear your optinion on this question. Many thanks :)

**Comment by skhu101** [comment]
Hi, thanks for your interest in our work. Based on my observation, the training converges for about 1~2 day. Longer training makes tiny differences in the final performance.
